lib/syncUserFollowers.js

```diff
@@ -0,0 +1,136 @@
+/*global Buffer*/
```

Comment:
Just a general question about this file. Is there any reason why we have this specific file in `kat-client-proxies` when it is only ever used in `kat-myft-syncher`? (From what I can see)

For ease of maintenance, I would imagine it would be nicer to have it in `kat-myft-syncher`, as any changes to this code requires we do another release of kat-client-proxies, and then update kat-myft-syncher to use this new release.

Reply:
For now I'd like to keep it where it is. In case we want to use it other that kat-myft-syncher